mllib/src/main/scala/org/apache/spark/mllib/optimization/NNLS.scala

```diff
@@ -53,8 +53,13 @@ private[spark] object NNLS {
    * projected gradient method.  That is, find x minimising ||Ax - b||_2 given A^T A and A^T b.
    *
    * We solve the problem
-   *   min_x      1/2 x^T ata x^T - x^T atb
-   *   subject to x &gt;= 0
+   *
+   * <blockquote>
+   *    $$
+   *    min_x 1/2 x^T ata x^T - x^T atb
+   *    $$
+   * </blockquote>
```

Comment:
- Java
  ![2016-11-28 2 32 13](https://cloud.githubusercontent.com/assets/6477701/20657212/799229f6-b577-11e6-9616-30a1e3f7ee1f.png)


- Scala (not found but manually built after changing the access modifier)
  ![2016-11-28 2 03 57](https://cloud.githubusercontent.com/assets/6477701/20657201/5a8d37a8-b577-11e6-8af5-8ed07c65a0ac.png)